README.md

```diff
@@ -17,6 +17,23 @@ With the Gemini CLI you can:
 - Ground your queries with the [Google Search](https://ai.google.dev/gemini-api/docs/grounding)
   tool, built in to Gemini.
 
+## Table of Contents
```

Comment:
@jackwotherspoon @jkcinouye whatchu think?

Reply:
@NTaylorMullen 

My 2 cents: I think a TOC is unnecessary here, the README is pretty short in general. For a longer README it would make sense, but I actually think the TOC on a short README hinders readability.